### ISSUE TITLE
Make whole project runnable with one single command

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#49]
## User story
1. As development team
2. I want to make my product easily executable with one single click
3. So Stackholders are able to easily build and execute the project and we save time in the development process

## Acceptance criteria
* It is possible to execute the Automated Data Pipeline to create the raw dataset with one command
* It is possible to execute the Automated Data Pipeline to create the Q&A dataset with one command
* It is possible to execute the whole project with one command
* It is documented how the pipelines and the project can be executed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
